examples/regression/1-quickstart/plot_prefit.py

```diff
@@ -79,11 +118,25 @@ def f(x: NDArray) -> NDArray:
     color="gray",
     ls="--",
 )
-plt.plot(X_test[order], y_pred[order], label="Prediction intervals")
-plt.fill_between(X_test[order], y_pred_low[order], y_pred_up[order], alpha=0.2)
+plt.plot(X_test[order], y_pred[order], label="Predictions")
+plt.fill_between(
+    X_test[order],
+    y_pred_low[order],
+    y_pred_up[order],
+    alpha=0.4,
+    label="prediction intervals QR"
+)
+plt.fill_between(
+    X_test[order],
+    y_pred_low_cqr[order],
+    y_pred_up_cqr[order],
+    alpha=0.4,
+    label="prediction intervals CQR"
+)
 plt.title(
-    f"Target and effective coverages for "
-    f"alpha={alpha}: ({1 - alpha:.3f}, {coverage:.3f})"
+    f"Target and effective coverages for:\n "
+    f"QR alpha={alpha}: ({1 - alpha:.3f}, {coverage:.3f})\n"
```

Comment:
QR stands for quantile regression ? If so, I'm not sure the term is correct as you just used a LGBM fitted with a QR and then applied a split-conformal method with MAPIE.

Reply:
changed